doc/source/user/tutorial-svd.rst

```diff
@@ -0,0 +1,440 @@
+================================================
+Tutorial: Linear algebra on n-dimensional arrays
+================================================
+
+.. currentmodule:: numpy
+
+.. testsetup::
+
+   import numpy as np
+   np.random.seed(1)
+
+**Prerequisites**
+
+Before reading this tutorial you should know a bit of Python. If you
```

Comment:
```suggestion
Before reading this tutorial, you should know a bit of Python. If you
```
Hi! I'm just making a few very small suggestions regarding language. Feel free to use or ignore them :)

Reply:
By the way, this tutorial is awesome!! Very nice work!